project/plugins.sbt

```diff
@@ -2,6 +2,12 @@
 //
 // e.g. addSbtPlugin("com.github.mpeltonen" % "sbt-idea" % "1.1.0")
 
+// Add EPFL Artifactory if on CI
+resolvers ++= {
```

Comment:
It might be. Maybe we should just include it in `Build.scala` and then update our cache. WDYT?